libraries/AP_Mission/AP_Mission.h

```diff
@@ -504,6 +512,7 @@ class AP_Mission {
     // parameters
     AP_Int16                _cmd_total;  // total number of commands in the mission
     AP_Int8                 _restart;   // controls mission starting point when entering Auto mode (either restart from beginning of mission or resume from last command run)
+    AP_Int8                _options;    // bitmask options for missions, currently for mission retention on reboot but can be expanded as required
```

Comment:
If this is going to be an options bit mask, I'd rather we store it in a AP_Int16 at least so that we have a bit more in the way of expansion available. (AP_Int32 is a bit overrated, as the float transport mechanism sets an early limit)

Reply:
I agree, 16 is better.